shared/src/components/Product/types.tsx

```diff
@@ -20,3 +24,40 @@ export interface HeaderScrollIndicatorProps {
   direction: "left" | "right";
   show: boolean;
 }
+
+export interface ProductCatalogueProps {
+  variant: "landing" | "webapp";
+  onVaultPress: (vault: VaultOptions) => void;
+}
+
+export type DesktopViewType = "grid" | "gallery";
+
+export const VaultStrategyList = ["COVERED-CALL", "PUT-SELLING"] as const;
+export type VaultStrategy = typeof VaultStrategyList[number];
+export const VaultStrategyMap: {
+  [vault in VaultOptions]: VaultStrategy;
+} = {
+  "rETH-THETA": "COVERED-CALL",
+  "rBTC-THETA": "COVERED-CALL",
+  "rUSDC-ETH-P-THETA": "PUT-SELLING",
+  "rUSDC-BTC-P-THETA": "PUT-SELLING",
+};
```

Comment:
We should move these to constants.ts because we will be updating this list quite a bit